### ISSUE TITLE
[Feature request] Difficulty in the  FSRS Stats

### ISSUE THREAD

**Issue by galantra** [open-spaced-repetition/fsrs4anki-helper#179]
I read the introduction of https://github.com/open-spaced-repetition/fsrs4anki

> The scheduler is based on a variant of the DSR (Difficulty, Stability, Retrievability) model, which is used to predict memory states

and concluded that perhaps it makes sense to add average difficulty to the Stats to complete the DSR triad?

**Comment by L-M-Sherlock** [comment]
The difficulty in FSRS is relative, not absolute. So I think it is meaningless to add average difficulty to the Stats.

**Comment by galantra** [comment]
Background: I noticed a discrepancy because in https://github.com/open-spaced-repetition/fsrs4anki-helper/commit/7b08c4e8989ebb7780c611804839a15a1e7af72c, +line 101, I referenced the [two component model of memory](https://supermemo.guru/wiki/Two_component_model_of_memory). But this reference can be misleading, since FSRS actually uses a three component model.

> The difficulty in FSRS is relative, not absolute. So I think it is meaningless to add average difficulty to the Stats.

I see. This means that you can't compare difficulty charts of one collection against another collection (especially not one of another user), right? 

Instead, you can make statements like these:

![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/52420923/8d21a669-1a67-4b57-98dd-32c8aa0cf82f)

This collection contains a lot of cards that are easy compared to the other cards in the collection.

![image](https://user-images.githubusercontent.com/83031600/239710716-7319276c-7293-43d9-8285-353f7c52a6b6.png)

This collection contains a lot of cards that are difficult compared to the other cards in the collection.

**Comment by Expertium** [comment]
If you Shift+Left Click stats and scroll all the way to the bottom, you will see the distribution of D.

**Comment by L-M-Sherlock** [comment]
By the way, it is possible to calculate the ease factor for cards.

**Comment by galantra** [comment]
> If you Shift+Left Click stats and scroll all the way to the bottom, you will see the distribution of D. I hope that satisfies your needs. On a side note, I wonder how many people even know about that chart. It feels like nobody ever scrolls down to find it.

Thanks, I didn't know or didn't remember that. Yes, it is more useful than average `D`.

We can add it to the Helper ReadMe. For example: [pull request]

I also added a description (definition? paraphrase?) of what D represents in plain English. However, I would actually prefer the following:

1. This kind of short paraphrases of R, S and D in plain language deserves a more visible place. Be it in the ReadMe of the main repo, be it as an introduction in the wiki. That's because (I guess) the page https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm looks recondite to user who are less well-versed in the Math.
(The Wiki home page now has links to Expertium's reddit posts, which is an improvement.)
2. The RSD paraphrases in the Helper ReadMe should be replaced with a simple link to the to-be-created paraphrases section in the main documentation.
3. The “Interpretation” section in Anki's stats screen should stay, since it explains more terms than D, S, R, and since it explains S and R for the specific context of the stats window.

**Comment by Expertium** [comment]
@L-M-Sherlock some small suggestions regarding the D graph:
1) Replace "Lower number = less difficult (easier), higher number = more difficult (harder)" with "Lower value of D (horizontal axis) = less difficult, higher value of D = more difficult".
2) Make the color of the bars a little darker.
3) Change the vertical axis so that it always starts from 0.

**Comment by L-M-Sherlock** [comment]
![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/32575846/9b18fbaa-6b92-4392-8984-03b85f3fcedd)

Done.
